node/cli2/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "node-cli2"
+version = "0.1.0"
+
+[dependencies]
+beacon = { path = "../../core/beacon" }
+clap = "2.32.0"
+client = { path = "../client" }
+network = { path = "../network", features = ["test-utils"] }
```

Comment:
test-utils in prod dependencies? probably should move the code that is not test utils out